### PR TITLE
Cl 1373 cl examples versions update 2023

### PR DESCRIPTION
**Pull request by Sl-violet**
This PR updates the examples to work with charting library version 23.043.

- Update READMEs to mention new minimum version
- Update versions of packages for:
- [x] android
- [x] angular
- [x] iOS - swift
- [x] nextjs-javascript
- [x] nuxtjs
- [x] react-javascript
- [ ] react-native
- [x] react-typescript
- [x] ruby-on-rails
- [ ] vuejs

### CODE REVIEW

**Review by romfrancois — APPROVED** [review]
All the changes look good to me.

**Comment by Sl-violet** [comment]
3 more added: angular, android, iOS-swift

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (romfrancois) [inline .gitignore:1]: Either to be removed or changed not to expose your username in it

Comment (romfrancois) [inline .gitignore:1]: Either to be removed or changed not to expose your username in it

**Review by romfrancois — APPROVED** [review]
LGTM

**Pull request merged**